wolox-books/src/app/screens/list-books/list-books.component.scss

```diff
@@ -0,0 +1,35 @@
+@import 'src/scss/color';
+@import 'src/scss/fonts';
+
+.container-cards {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  padding: 50px 300px;
+}
+
+.content-card {
+  background-color: $white ;
+  box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
+  height: 320px;
+  padding: 28px;
+  margin: 30px 20px;
+  width: 198px;
+}
+
+.img-cards {
+  object-fit: cover;
+  padding-bottom: 15px;
```

Comment:
If the padding is supposed to add space between image and content, you should use margin